inst/docker/curatedMetagenomics/curatedMetagenomicData_pipeline.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+### usage: bash curatedMetagenomicData_pipeline.sh sample_name "SRRxxyxyxx;SRRyyxxyyx"
+
+### before running this script, be sure that these tools are in your path
+# fastq-dump
+# humann2
+# metaphlan2
+# python
+
+### before running this script, set these paths and variables
+pm=/opt/metaphlan2/biobakery-metaphlan2/metaphlan2.py #metaphlan2 path (like /tools/metaphlan2/bin/metaphlan2.py)
+pc=/dbs/humann2/chocophlan # chocophlan database (nucleotide-database for humann2, like /databases/chocophlan
+pp=/dbs/humann2/uniref # uniref database (protein-database for humann2, like /databases/uniref)
```

Comment:
Maybe I'm being dense here, but where is the uniref database chosen, e.g. following your issue #98 ?

Reply:
Right now, it is uniref50. I am going to extract the databases and make those "inputs" to the pipeline that can be specified at runtime. In cloud world, file transfers from cloud storage to instances is pretty performant, so we can actually pull it in for each sample.